### PR TITLE
added 4.0.0 alpha1 release notes

### PR DESCRIPTION
**Pull request by rafaelmartins**
Signed-off-by: Rafael Martins <rmartins@redhat.com>

### CODE REVIEW

File: source/release/4.0.0/index.html.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+title: OVirt 4.0.0 Release Notes
+category: documentation
+authors: rafaelmartins
+---
+
+# oVirt 4.0.0 Release Notes
+
+The oVirt Project is pleased to announce the availability of oVirt 4.0.0 alpha1 release as of April 5th, 2016.
+
+oVirt is an open source alternative to VMware vSphere, and provides an awesome KVM management interface for multi-node virtualization. This release is available now for Red Hat Enterprise Linux 6.7, CentOS Linux 6.7 (or similar) and Red Hat Enterprise Linux 7.2, CentOS Linux 7.2 (or similar).
```
Comment (mykaul) [inline source/release/4.0.0/index.html.md]: Do we support 6.7 (on the engine side) ? I think it's unclear here.

File: source/release/4.0.0/index.html.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+For a general overview of oVirt, read [the Quick Start Guide](Quick_Start_Guide)
+and the [about oVirt](about oVirt) page.
+
+## Install / Upgrade from previous versions
+
+### Fedora / CentOS / RHEL
+
+## ALPHA RELEASE
+
+In order to install oVirt 4.0.0 Alpha Release you've to enable oVirt 4.0.0 alpha1 release repository.
```
Comment (mykaul) [inline source/release/4.0.0/index.html.md]: you've -> you will need

**Comment by mykaul** [comment]
I'm missing some comment about what is an alpha release, not to be used in production, not feature complete, we'll be happy to get feedback (and how to provide it), etc.

File: source/release/4.0.0/index.html.md (hunk 1 above)
Comment (sandrobonazzola) [inline source/release/4.0.0/index.html.md]: No, in 4.0 we're not supporting el6 anymore

**Comment by sandrobonazzola** [comment]
I think you can adapt alpha statement from Fedora: https://getfedora.org/en/workstation/prerelease/

**Comment by rafaelmartins** [comment]
Updated based on comments, also added list of fixed bugs

**Comment by sandrobonazzola** [comment]
:+1:

**Pull request merged**